packages/edit-site/src/components/sidebar/navigation-menu-sidebar/style.scss

```diff
@@ -0,0 +1,42 @@
+@keyframes loadingpulse {
```

Comment:
We have similar code elsewhere ([packages/block-editor/src/components/link-control/style.scss](https://github.com/WordPress/gutenberg/blob/65f7c31da4b5e0e89ab8b3b6997361834ddd0b00/packages/block-editor/src/components/link-control/style.scss)) so maybe we should make this a mixin?